bookstore-actix/migration/src/m20220101_000001_create_table.rs

```diff
@@ -0,0 +1,35 @@
+use sea_orm_migration::prelude::*;
+use sea_orm_migration::sea_orm::ConnectionTrait;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let db = manager.get_connection();
+
+        // Use `execute_unprepared` if the SQL statement doesn't have value bindings
+        db.execute_unprepared(
+            "CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";
```

Comment:
Do we need this extension ? I think the latest versions of postgres support uuid out of the box

Reply:
Looks like we do ... 
But anyway the extension is only created if it does not exist.

Reply:
I think you could use gen_random_uuid(). That is active by default. It would make sure we use the same one than in the other services. `uuid-ossp` offers some more functions for special types of UUIDs.

Reply:
They are basically not much different (one also uses network infos) and have a similar runtime behavior. For the sake of the benchmark this will make no difference compared to all the other "real" differences in the various frameworks and technologies.

Reply:
I just wanted to point out why the Quarkus impl didn't need the extension ;)